Assets/Scripts/PlayerController.cs

```diff
@@ -1,30 +1,47 @@
 ﻿using UnityEngine;
 using System.Collections;
+using UnityEngine.UI;
 
-public class PlayerMovement : MonoBehaviour
+public class PlayerController : MonoBehaviour
 {
+    public float maxJumps; // maximum number of jumps allowed
     public float speed; // ball speed
     public float thrust; // the force of the jump
+    public int playerHealth = 100;
+    public float secondsUntilRespawn;
+    public PointCounter pointCounter;
+    public int flashNumber;
+    public ParticleSystem damageParticles;
+    public GameObject retryButton;
+
     private float moveInput; // horizontal axis value for input
     private Rigidbody2D ballRigidBody; // rigid body (private to only access it from the script) for the player ball
-    public float maxJumps; // maximum number of jumps allowed
+    private Transform ballTransform;
     private float remainingJumps; // number of jumps remaning
-    public int playerHealth = 100;
     private int environmentDamage = 50;
     private int lethalDamage = 999;
-    public float secondsUntilRespawn;
-    public PointCounter pointCounter;
+    private bool isFacingRight = true;
+    private IEnumerator damageEffectsLoop;
+
```

Comment:
Remove the spaces. only one space should remain between `private IEnumerator damageEffectsLoop;` and `void Start()`